src/main/resources/application.yml

```
@@ -168,6 +168,24 @@ workflow:
     retry-backoff-coefficient: "\${DEBT_POSITION_ACTIVITY_CONFIG_RETRY_BACKOFF_COEFFICIENT:\${workflow.default.retryBackoffCoefficient}}"
     retry-maximum-attempts: "\${DEBT_POSITION_ACTIVITY_CONFIG_RETRY_MAXIMUM_ATTEMPTS:\${workflow.default.retryMaximumAttempts}}"
 
+  organization-payments-reporting-pago-pa:
```

this should be aligned to the config set on `OrganizationPaymentsReportingPagoPaFetchWfConfig`
```suggestion
  payments-reporting-pagopa-organization-fetch:
```

ps: fix ENV variable using prefix:
ORGANIZATION_PAYMENTS_REPORTING_PAGOPA_FETCH